src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFoodForm';
+import SearchBar from './components/SearchBar';
+import { Row, Col } from 'antd'
 
 function App() {
+  const [allFoods, setAllFoods] = useState(foods)
+  
```

Comment:
For the Search function to work properly you need to add a new `useState` like this:
`  const [showFoods, setShowFoods] = useState(foods);`